model/patch/patch.go

```diff
@@ -172,6 +172,8 @@ type Patch struct {
 	// MergeStatus is only used in gitServePatch to send the status of this
 	// patch on the commit queue to the agent
 	MergeStatus string `json:"merge_status"`
+	// MergedFrom is only used when a merge patch is created from an existing patch.
```

Comment:
Can this specify that the value is set to the patch ID of the existing patch?

Reply:
done